tests/patterns/rust/attribute_matching.rs

```diff
@@ -0,0 +1,30 @@
+// MATCH:
+#[test]
+#[more(complicated, attribute)]
+fn foo1() {
+}
+
+#[not_test]
+#[more(complicated, attribute)]
+fn foo1() {
+}
+
+#[test]
+fn fooToo() {
+}
+
+// MATCH:
+#[test]
```

Comment:
If the order is reversed, should a match be produced? Can we add a test case exhibiting this behavior?

Reply:
Added a test case to demonstrate.